src/App.css

```diff
@@ -38,17 +38,20 @@
   overflow-y: auto;
 }
 .type:before {
-  content: "Type: "
+  content: "Type: ";
```

Comment:
I think it is better to type this in the HTML not in the styles. content: attribute is usually used for buttons and spans, where you want to add some kind of a font image or symbol